cmd/installer/cli/reset.go

```diff
@@ -330,25 +330,55 @@ type etcdMembers struct {
 
 // leaveEtcdcluster uses k0s to attempt to leave the etcd cluster
 func (h *hostInfo) leaveEtcdcluster() error {
-
-	// if we're the only etcd member we don't need to leave the cluster
-	out, err := helpers.RunCommand(k0sBinPath, "etcd", "member-list")
-	if err != nil {
-		return err
+	// Try to list members with retries
+	var memberlist etcdMembers
+	var out string
+	var err error
+
+	// Retry member list up to 3 times
+	for i := 0; i < 3; i++ {
+		out, err = helpers.RunCommand(k0sBinPath, "etcd", "member-list")
+		if err == nil {
+			err = json.Unmarshal([]byte(out), &memberlist)
+			if err == nil {
+				break
+			}
+		}
+		if i < 2 { // Don't sleep on last attempt
+			time.Sleep(2 * time.Second)
+		}
 	}
-	memberlist := etcdMembers{}
-	err = json.Unmarshal([]byte(out), &memberlist)
+
 	if err != nil {
-		return err
+		logrus.Warnf("Unable to list etcd members, continuing with reset: %v", err)
```

Comment:
this used to return an error before, is this intentional?

Reply:
Yes - we can still reset the actual node at that point and so we should